README.md

```diff
@@ -29,19 +29,31 @@ See a complete working example in the [roots-example-project.com repo](https://g
 
 ## Installation
 
-1. Create a new project - `composer create-project roots/bedrock`
-2. Copy `.env.example` to `.env` and update environment variables:
+1. Create a new project in a new folder for your project:
+
+  `composer create-project roots/bedrock your-project-folder-name`
+
+1. Copy `.env.example` to `.env` and update environment variables:
   * `DB_NAME` - Database name
   * `DB_USER` - Database user
   * `DB_PASSWORD` - Database password
   * `DB_HOST` - Database host
   * `WP_ENV` - Set to environment (`development`, `staging`, `production`)
   * `WP_HOME` - Full URL to WordPress home (http://example.com)
   * `WP_SITEURL` - Full URL to WordPress including subdirectory (http://example.com/wp)
-  * `AUTH_KEY`, `SECURE_AUTH_KEY`, `LOGGED_IN_KEY`, `NONCE_KEY`, `AUTH_SALT`, `SECURE_AUTH_SALT`, `LOGGED_IN_SALT`, `NONCE_SALT` - Generate with [wp-cli-dotenv-command](https://github.com/aaemnnosttv/wp-cli-dotenv-command) or from the [Roots WordPress Salt Generator](https://roots.io/salts.html)
-3. Add theme(s) in `web/app/themes` as you would for a normal WordPress site.
-4. Set your site vhost document root to `/path/to/site/web/` (`/path/to/site/current/web/` if using deploys)
-5. Access WP admin at `http://example.com/wp/wp-admin`
+  * `AUTH_KEY`, `SECURE_AUTH_KEY`, `LOGGED_IN_KEY`, `NONCE_KEY`, `AUTH_SALT`, `SECURE_AUTH_SALT`, `LOGGED_IN_SALT`, `NONCE_SALT`
+
+  If you want to automatically generate the security keys (assuming you have wp-cli installed locally) you can use the very handy [wp-cli-dotenv-command][wp-cli-dotenv]:
+
+      wp package install aaemnnosttv/wp-cli-dotenv-command
+
+      wp dotenv salts regenerate
+
+  Or, you can cut and paste from the [Roots WordPress Salt Generator][roots-wp-salt].
+
+1. Add theme(s) in `web/app/themes` as you would for a normal WordPress site.
+1. Set your site vhost document root to `/path/to/site/web/` (`/path/to/site/current/web/` if using deploys)
+1. Access WP admin at `http://example.com/wp/wp-admin`
```

Comment:
I think it should still be 1., 2., 3., etc. Otherwise it looks stupid when it's not rendered. The note regarding the use of `1.` is strictly for those who are using the "lazy list numbering."

> The point is, if you want to, **you can use ordinal numbers in your ordered Markdown lists, so that the numbers in your source match the numbers in your published HTML.** But **if you want to be lazy,** you don’t have to.

Sweet, we can choose between being lazy or not being lazy. Doesn't sound like either one is "recommended" or "instructed."

> **If you do use lazy list numbering,** however, you should still start the list with the number 1. At some point in the future, Markdown may support starting ordered lists at an arbitrary number.

This isn't an issue we need to worry about if we're not lazy.

So let's not be lazy.